### PR TITLE
Collator protocol debug

### PR DESCRIPTION
**Pull request by vstakhov**
This PR is intended to add some missing debug messages to the collator protocol part.

### CODE REVIEW

File: node/network/collator-protocol/src/validator_side/mod.rs [hunk 1]
```diff
@@ -662,6 +662,14 @@ async fn fetch_collation<Context>(
 
 	if state.peer_data.get(&peer_id).map_or(false, |d| d.has_advertised(&relay_parent)) {
 		request_collation(ctx, state, relay_parent, para_id, peer_id, tx).await;
+	} else {
```
Comment (eskimor) [inline node/network/collator-protocol/src/validator_side/mod.rs:665]: This does also trigger if there is no peer data for that peer.

File: node/network/collator-protocol/src/validator_side/mod.rs [hunk 2]
```diff
@@ -883,6 +909,12 @@ async fn process_incoming_peer_message<Context>(
 
 			let peer_data = match state.peer_data.get_mut(&origin) {
 				None => {
+					gum::debug!(
+						target: LOG_TARGET,
+						peer_id = ?origin,
+						?relay_parent,
+						"Unknown peer",
```
Comment (eskimor) [inline node/network/collator-protocol/src/validator_side/mod.rs:916]: maybe more descriptive: "Message received from unknown peer"

File: node/network/collator-protocol/src/validator_side/mod.rs (hunk 1 above)
Comment (vstakhov) [reply node/network/collator-protocol/src/validator_side/mod.rs:675]: Yes, I think this message shoul be on the `warn!` level as it is quite unlikely.

**Review by rphmeier — APPROVED** [review]

**Comment by rphmeier** [comment]
bot merge

**Pull request merged**